config/jobs/kubernetes/sig-testing/integration.yaml

```diff
@@ -72,6 +72,11 @@ presubmits:
           requests:
             cpu: 8
             memory: 20Gi
+      tolerations:
```

Comment:
don't we also need a node selector? otherwise they might schedule to the other node pool?

Reply:
I don't see a taint on the C4 nodepool either

Reply:
selector: https://github.com/kubernetes/test-infra/pull/34840